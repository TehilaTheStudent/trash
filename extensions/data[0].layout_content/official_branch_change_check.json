[
  {
    "componentName": "d-form",
    "id": 1685023016672,
    "category": "form",
    "props": {
      "layout": "vertical",
      "ref": "formRef"
    },
    "children": [
      {
        "componentName": "d-form-item",
        "id": 1685023018114,
        "category": "form",
        "props": {
          "label": "Target Branch",
          "field": "TARGET_COMPARE_BRANCH",
          "help-tips": "Specify the branch name as the target branch for the check. The pipeline running fails if the current pipeline running branch lags behind the specified branch.. Support fixed parameters such as master or environment variable references.",
          "rules": [
            {
              "required": true,
              "message": "单行输入不能为空",
              "trigger": [
                "blur",
                "change"
              ]
            },
            {
              "regex": "",
              "message": "单行输入内容不符合规范",
              "trigger": [
                "blur",
                "change"
              ]
            }
          ],
          "visible": {
            "type": "fixed",
            "conditions": [],
            "value": true
          },
          "disabled": false
        },
        "children": [
          {
            "componentName": "d-input",
            "id": 1685023018113,
            "props": {
              "inputType": "normal",
              "isTrim": true,
              "autocomplete": "off",
              "defaultValue": "",
              "disabled": {
                "type": "fixed",
                "conditions": [],
                "value": false
              }
            },
            "category": "form"
          }
        ]
      }
    ]
  }
]