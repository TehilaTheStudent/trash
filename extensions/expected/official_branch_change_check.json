{
  "plugin_name": "official_branch_change_check",
  "display_name": "",
  "data": [
    {
      "unique_id": "2c70a2e74162415cb94f40917b20f36e",
      "name": "TARGET_COMPARE_BRANCH",
      "default_value": "",
      "plugin_name": "official_branch_change_check",
      "version": "0.0.1",
      "type": "d-input",
      "workspace_id": "*",
      "layout_content": "[{\"componentName\":\"d-form\",\"id\":1685023016672,\"category\":\"form\",\"props\":{\"layout\":\"vertical\",\"ref\":\"formRef\"},\"children\":[{\"componentName\":\"d-form-item\",\"id\":1685023018114,\"category\":\"form\",\"props\":{\"label\":\"Target Branch\",\"field\":\"TARGET_COMPARE_BRANCH\",\"help-tips\":\"Specify the branch name as the target branch for the check. The pipeline running fails if the current pipeline running branch lags behind the specified branch.. Support fixed parameters such as master or environment variable references.\",\"rules\":[{\"required\":true,\"message\":\"单行输入不能为空\",\"trigger\":[\"blur\",\"change\"]},{\"regex\":\"\",\"message\":\"单行输入内容不符合规范\",\"trigger\":[\"blur\",\"change\"]}],\"visible\":{\"type\":\"fixed\",\"conditions\":[],\"value\":true},\"disabled\":false},\"children\":[{\"componentName\":\"d-input\",\"id\":1685023018113,\"props\":{\"inputType\":\"normal\",\"isTrim\":true,\"autocomplete\":\"off\",\"defaultValue\":\"\",\"disabled\":{\"type\":\"fixed\",\"conditions\":[],\"value\":false}},\"category\":\"form\"}]}]}]",
      "front_data_prop": null,
      "label": null,
      "description": null,
      "required": null,
      "extend_prop": {
        "api_options": null,
        "api_type": null,
        "show_placeholder": null,
        "options": null,
        "disabled_conditions": null,
        "visible_conditions": null
      },
      "validation": {
        "required_message": null,
        "regex": null,
        "regex_message": null,
        "max_length": null,
        "min_length": null
      }
    }
  ]
}